src/core.c/Array.rakumod

```diff
@@ -1093,6 +1093,57 @@ my class Array { # declared in BOOTSTRAP
                :range("0..^{self.elems - $offset}")
              ).throw
     }
+    #------ splice(offset,size,positional scalar) candidates
```

Comment:
would prefer the `Array.splice` changes to be in a separate PR, after it has become clear that the `Positional $foo` feature works and has spectests.  This would allow for better bisectability should problems with `Array.splice` becime clear after a blin run.

Reply:
They are already separate commits, but if you feel a second PR is better I don't have a problem with that.

Reply:
The PR is now adjusted to only address the positional scalar disambiguation.